lcviewernoqt/CMakeLists.txt

```diff
@@ -42,6 +62,20 @@ drawables/dragpoints.cpp
 drawables/tempentities.cpp
 drawables/CursorLocation.cpp
 drawitems/lcvinsert.cpp
+viewersettings.cpp
+${PROJECT_SOURCE_DIR}/painters/opengl/SOURCES/vertexbuffer.cpp
```

Comment:
It's better to not keep a `SOURCES` folder here, and keep the .h and .cpp in `opengl` folder.
I don't see a problem for `RES` folder, but keep names lowercase please.

Don't forget to use a good IDE to do this in few minutes